community/contributing/docs/reviewing-prs.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: single
+title: Reviewing Pull Requests
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Review a pull request
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/spinnaker/spinnaker.github.io/pulls) section in the Spinnaker website repository to see a list of all currently open pull requests.
+
+Reviewing documentation pull requests is a great way to get started contributing to the Spinnaker community. It helps you to learn the code base and builds rapport with other contributors.
+
+# Before you begin
+
+Before reviewing documentation, do the following:
+
+Read the [documentation style guide](https://www.spinnaker.io/community/contributing/docs/docs-style-guide/) so that you can leave informed feedback.
+
+1. Review the [code of conduct](https://github.com/spinnaker/spinnaker.github.io/blob/master/community/contributing/code-of-conduct.md) and ensure that you abide by it at all times.
+2. Be polite, helpful, and considerate of others.
+3. Comment on positive aspects of a PR, as well as changes.
+4. Be empathetic and mindful of how your changes or review may be received.
+5. Ask clarifying questions if something in a PR is unclear.
+6. Assume positive intent.
+7. If you are an experienced contributor, consider pairing with new contributors whose work requires extensive changes.
+8. Be mindful of the needs of contributors that are neurodivergent or have a learning disability. Ask your fellow contributors how they work best and what communication style they prefer.
+
+# Start the review process
+
+1. Go to (<https://github.com/spinnaker/spinnaker.github.io/pulls>) to see a list of all current open PRs in the Spinnaker repository.
+
+2. Filter the open PRs using one or all of the following labels:
+
+  - Beginner friendly
+  - Help wanted
+  - Questions
+  - no-lifecycle
+
+    If a PR is labelled `draft PR` it means that the contributor would like feedback on their PR throughout the review process. Reviewers should encourage new contributors to use **draft PRs** rather than open a pull request.
+
+  - Once you've found a PR to review, understand the change by:
```

Comment:
So you mention 'PR' but I don't see where you mentioned pull request (PR)... so suggest changing line 1:

`Anyone can review a documentation pull request (PR)`